hera_qm/firstcal_metrics.py

```diff
@@ -312,12 +312,33 @@ def __init__(self, calfits_file):
         self.UVC = UVCal()
         self.UVC.read_calfits(calfits_file)
 
+        if len(self.UVC.jones_array) > 1:
+            raise ValueError('Sorry, only single pol firstcal solutions are '
+                             'currently supported.')
+        pol_dict = {-5: 'x', -6: 'y'}
+        try:
+            self.pol = pol_dict[self.UVC.jones_array[0]]
+        except KeyError:
+            raise ValueError('Sorry, only calibration polarizations "x" and '
+                             '"y" are currently supported.')
+
+        # Get the firstcal.rotated_antenna.metric file
+        if self.UVC.cal_type == 'gain':
+            with open(calfits_file + '.rotated_metric.json') as rot_file:
+                rmetric = json.load(rot_file)
+            self.delays = np.array([rmetric['delays'][str(ai)] for ai in self.UVC.ant_array]).squeeze()
+            self.rotated_antennas = { int(k): self.pol for k in rmetric['rotated_antennas'] }
```

Comment:
Is there a way to add a check that the pol from the calfits file is the same as the pol for the rotated ants? Even better would be a check that they're the same observation.